guides/09_Amazon_Voice_Focus.md

```diff
@@ -285,7 +285,23 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 Enabling Voice Focus with Echo Reduction is a two step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Create a meeting with support for Echo Reduction using Amazon Chime Meeting SDK namespace
+
+To create meetings, the following line of code is used to address the namespace:
```

Comment:
I'm sure that this section will confuse many builders because we haven't used terms like **Amazon Chime SDK Meetings** often in our GitHub docs. Also, builders can use AWS CLI or REST API to create a meeting, while your section only explains the AWS SDK for JavaScript usage.

Instead of AWS SDK for JavaScript examples, I recommend adding a warning section with a link to the [Migrating to the Amazon Chime SDK Meetings namespace](https://docs.aws.amazon.com/chime/latest/dg/meeting-namespace-migration.html) at the top.

1. Add this warning at the top of this section.

> ⚠️ To use the Echo Reduction feature, you must migrate to the [Amazon Chime SDK Meetings](https://docs.aws.amazon.com/chime/latest/APIReference/API_Operations_Amazon_Chime_SDK_Meetings.html) namespace. To do so, you can follow the steps in the [Migrating to the Amazon Chime SDK Meetings namespace](https://docs.aws.amazon.com/chime/latest/dg/meeting-namespace-migration.html).

2. Remove two examples and namespace explanation (line 302).

3. Add your constructor creation to the existing example at line 309.

```
// You must migrate to the Amazon Chime SDK Meetings namespace.
const chime = AWS.ChimeSDKMeetings({ region: "eu-central-1" });

// Create meeting 
const meetingInfo = await chime.createMeeting({
  ...
  MeetingFeatures: {
    Audio: {
      EchoReduction: 'AVAILABLE' 
    }
  } 
}).promise();
```

Reply:
Following this method. Thank you so much!